package.json

```diff
@@ -912,7 +912,7 @@
   },
   "optionalDependencies": {
     "@folio/plugin-create-inventory-records": "^3.0.0",
-    "@folio/plugin-find-instance": "^6.0.0"
+    "@folio/plugin-find-instance": "^7.0.0"
```

Comment:
Please update the changelog.

Reply:
I made this change as a second part of UIIN-2508 and it has an entry already